spoon-runner/src/main/java/com/squareup/spoon/SpoonUtils.java

```diff
@@ -129,17 +130,19 @@ static void createAnimatedGif(List<File> testScreenshots, File animatedGif) thro
   }
 
   private static void waitForAdb(AndroidDebugBridge adb) {
-    for (int i = 1; i < 10; i++) {
+    long timeOut = 30000; // 30 sec
```

Comment:
kill the comment and replace with `TimeUnit.SECONDS.toMillis(30)`.  Same for the next line.  And it's not a hard requirement but I'd rename them to `timeOutMs` and `sleepTimeMs`.